overlord/hookstate/hookmgr.go

```diff
@@ -142,6 +147,14 @@ func (m *HookManager) Stop() {
 	m.runner.Stop()
 }
 
+func (m *HookManager) hijacked(snapName, hookName string) HookFunc {
+	return m.hijackedMap[fmt.Sprintf("%s:%s", snapName, hookName)]
+}
+
+func (m *HookManager) RegisterHijacked(snapName, hookName string, f HookFunc) {
```

Comment:
I have mixed feeling about this interface:

* the asymmetry vs the Register interface is a bit strange, I would at least switch the order of hookName, snapName
* otoh a symmetrical interface would be much more complicated

:/

Reply:
I switched the order now, but yeah, having a RegsterHijacked with the same signature as Register() is probably tricky.